src/form.js

```diff
@@ -802,7 +802,32 @@ define([
             }
         },
         changeMugType: function (mug, questionType) {
-            this.mugTypes.changeType(mug, questionType);
+            var _this = this;
```

Comment:
Yeah, not necessary, but I like using `_this` as a habit. Otherwise I get bitten when making code more complex later on.